src/components/GracePeriodError/GracePeriodError.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ReactComponent as AlertIcon } from 'assets/images/errors/alert.svg';
+import { Trans } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import {
+  GracePeriodErrorStyled,
+  WarningMessageStyled,
+  IconStyled
+} from './GracePeriodErrorStyled';
+
+const GracePeriodError = () => {
+  const currentPlan = useSelector(state => state.planDetails.currentPlan);
```

Comment:
just wondering: if only PaymentInfo component will be embedded, this GracePeriodError will never be shown. What in case if PaymentInfo component and PlanDetails will be embedded on the same page? Will it work on PaymentInfo then?